File: Pset1_-_C/mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    //declare global variables
    int h, i, j;
    //input height of pyramid from user until a number from 1 to 8 is picked
    do
    {
        h = get_int("Height: ");
    }
    while (h > 8 || h <= 0);

    //RIGHT-ALIGNED TRIANGLE
    //iterate through rows
    for (i = 0; i < h; i++)
    {
        //insert spaces during interation
        for (j = i; j < h - 1; j++)
        {
            printf(" ");  
        }  
        //print # for the input height
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }
//insert space after every iteration    
printf("\n");
    }
}

File: Pset6_-_Python/mario.py
from cs50 import get_int

# get height input from user
height = get_int("Height: ")

# height bounds
while height < 0 or height == 0 or height > 8:
    height = get_int("Height: ")

#iterate through height
for i in range(1, height + 1):
    # insert spaces before hashes
    for j in range(height - i):
        print(" ", end="")
    for j in range(height - i, height):
        print("#", end="")
    height - 1
    # add new line
    print()
